bin/bootstrax

```diff
@@ -538,11 +544,19 @@ def infer_target(rd):
 
     n_fails = rd['bootstrax'].get('n_failures', 0)
     if n_fails:
-        log.debug(f'override_target::\tdeleting targets')
+        log.debug(f'Deleting targets')
         for _rm_t, _rm_t_after in remove_target_after_fails.items():
-            _t = [__t for __t in _t if (_rm_t not in __t and n_fails < _rm_t_after)]
-            _pp = [__pp for __pp in _pp if (_rm_t not in __pp and n_fails < _rm_t_after)]
+            for __i, __t in enumerate(_t):
```

Comment:
I think it's worth cleaning up the variable names here, `i` is cleaner than `__i`, and `potential_targets` is cleaner than `_t`. This isn't the c++ standard library, I don't think we need _this_ many underscores.

Reply:
>Honestly, this is not the prettiest part of the code. Will do some work on it when we add some new infer-logic for the linked mode as well as NV/MV processing.

yeah, this whole thing got messier than it supposed to